types/params.go

```diff
@@ -358,10 +433,27 @@ func (params ConsensusParams) Update(params2 *cmtproto.ConsensusParams) Consensu
 			res.Synchrony.Precision = *params2.Synchrony.GetPrecision()
 		}
 	}
+
+	if params2.Feature != nil {
+		// TODO: move ABCI FeatureParams
+		if params2.Feature.PbtsEnableHeight != nil {
+			res.Feature.PbtsEnableHeight = &params2.Feature.GetPbtsEnableHeight().Value
+		}
+	}
 	return res
 }
 
 func (params *ConsensusParams) ToProto() cmtproto.ConsensusParams {
+	feature := cmtproto.FeatureParams{}
+	if params.Feature.PbtsEnableHeight != nil {
+		feature.PbtsEnableHeight = &gogo.Int64Value{}
+		feature.PbtsEnableHeight.Value = *params.Feature.PbtsEnableHeight
```

Comment:
Go not being nice again. No better way to initialize a `Int64Value`?

Reply:
Agreed. Couldn't find any.